tests/unit/test_config.py

```diff
@@ -13,6 +13,11 @@ def _assert_config_read(environment, mock_config):
         mock_instance = mock_config.return_value
         Config.CONFIG = None  # Force config file reload
         prev_environment = os.environ.get(environment)
+        for to_remove in ['APPDATA', 'HOME', 'XDG_CONFIG_HOME']:
+            try:
+                del os.environ[to_remove]
```

Comment:
Yes, good catch. For some reason I thought I was restoring the entire environment, not just the one setting. Oops.